Cargo.toml

```diff
@@ -14,3 +18,8 @@ members = [
   "test_bin",
 ]
 exclude = ["test_utils"]
+
+[profile.release]
```

Comment:
This can't be good. This means that every panic in the whole project will abort instead of unwinding the stack. I don't think that's what we want

Reply:
@sphinxc0re this is pulled from existing config

Reply:
But this shouldn't be in the root `Cargo.toml`. Every wasm project should define it's own config